openedx/stanford/djangoapps/auth_lagunita/models.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Record extra lagunita-centric user data
+"""
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models
+
+USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
+
+
+class Info(models.Model):
+    """
+    Record extra fields that will be saved when a user registers
+
+    requested_marketing_optin: Track whether user has requested to
+        receive additional marketing messages
+    submitted_marketing_optin: Track whether system has submitted an
+        opt-in for user to receive additional marketing messages
+    """
+    user = models.OneToOneField(
+        USER_MODEL,
+    )
+    requested_marketing_optin = models.BooleanField(
+        default=False,
+    )
+    submitted_marketing_optin = models.BooleanField(
+        default=False,
+    )
+    created_date = models.DateTimeField(
+        auto_now_add=True,
+    )
+    modified_date = models.DateTimeField(
+        auto_now=True,
+    )
+
+    def __unicode__(self):
```

Comment:
`pylint` wanted to see this method.

The output doubles as the python command to perform a lookup of that object.